packages/roosterjs-editor-plugins/lib/plugins/Picker/PickerPlugin.ts

```diff
@@ -132,6 +132,8 @@ export default class PickerPlugin<T extends PickerDataProvider = PickerDataProvi
         this.currentInputLength = null;
         this.newInputLength = null;
         this.dataProvider.onDispose();
+        this.dataProvider = undefined;
```

Comment:
nit, this may cause problem when we go to strict mode.

You can declare another local var to allow undefined and assign dataProvider to it. Then do null check when we use it.